### PR TITLE
fix(cypress): backwards compat for version < 10.9.0

### PR DESCRIPTION
**Pull request by emmenko**
Leftover of #2830 

image

### CODE REVIEW

File: packages/cypress/src/add-commands/login.ts [hunk 1]
```diff
@@ -124,12 +130,18 @@ function loginByForm(commandOptions: CommandLoginOptions) {
     // For backwards compatibility.
     if (Cypress.config('experimentalSessionAndOrigin')) {
       // https://www.cypress.io/blog/2021/08/04/authenticate-faster-in-tests-cy-session-command/
-      cy.session(sessionKey, authCallback, {
-        cacheAcrossSpecs:
-          typeof commandOptions.disableCacheAcrossSpecs === 'boolean'
-            ? !commandOptions.disableCacheAcrossSpecs
-            : true,
-      });
+      cy.session(
```
Comment (CarlosCortizasCT) [inline packages/cypress/src/add-commands/login.ts:133]: I was thinking if we should also check the version for using the `cy.session` command.
It seems our `cypress` package has support from Cypress [v8.0](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/cypress/package.json#L49) and this command was introduced in [v8.2](https://docs.cypress.io/guides/references/changelog#8-2-0).

**Review by CarlosCortizasCT — APPROVED** [review]
👌

File: packages/cypress/src/add-commands/login.ts (hunk 1 above)
Comment (tdeekens) [reply packages/cypress/src/add-commands/login.ts:133]: Good point. In an ideal case yes. In a slightly pragmatic case maybe not. All internal repos use Cypress v10. I am not aware of a customer actually using exactly 8.0.0 (or the package at all). So maybe close the eye over this (at least from my view).

**Review by tdeekens — APPROVED** [review]

File: packages/cypress/src/add-commands/login.ts (hunk 1 above)
Comment (emmenko) [reply packages/cypress/src/add-commands/login.ts:133]: That's already covered by the check `Cypress.config('experimentalSessionAndOrigin')`.

In v8 this flag was called differently.

**Pull request merged**